### ISSUE TITLE
v6.0.0 public/.htaccess uses old rule syntax

### ISSUE THREAD

**Issue by vapier** [grokability/snipe-it#11071]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

with apache-2.4.53 and snipeit-6.0.0:
```
$ tail -n1 /var/log/apache2/error.log
[Fri May 13 20:13:37.406524 2022] [core:alert] [pid 394] [client ...:43204] /var/www/snipe-it/public/.htaccess: Invalid command 'Deny', perhaps misspelled or defined by a module not included in the server configuration
```

### Reproduction steps

1. use apache-2.4
2. install snipeit-6.0.0
3. fail

### Expected behavior

do not fail

### Snipe-IT Version

6.0.0

### Operating System

Debian Linux

### Web Server

apache

### PHP Version

7.8

**Comment by snipe** [comment]
I don't know what this means. We haven't changed the htaccess syntax in ages.

**Comment by snipe** [comment]
If you have a fix for this, we'd happily look at a PR, but we have to make sure it degrades well for older versions of Apache.

**Comment by vapier** [comment]
it means v6.0.0 is broken on apache-2.4 as the error log shows.  the broken change doesn't exist in the 5.x series, it was merged into the 6.x series.  so it's been broken in the 6.x series since commit 494ed3d17aaee68ae885d87948111ab299549094.

compare the use of Require-vs-Deny in the top-level .htaccess and public/.htaccess.

**Comment by snipe** [comment]
We'd welcome a PR to resolve this issue.

**Comment by snipe** [comment]
If you install `access_compat_module` it should work again.

**Comment by bby-bishopclark** [comment]
consider 

```
<Files "web.config">
  <IfModule mod_authz_core.c>
    Require all denied
  </IfModule>
  <IfModule !mod_authz_core.c>
    Order Deny,Allow
    Deny from All
 </IfModule>
</Files>
```

It's a common format for sussing versions.

**Comment by bby-bishopclark** [comment]
> compare the use of Require-vs-Deny in the top-level .htaccess and public/.htaccess.

Silly me -- didn't read it all.  Vapier's pointer to https://github.com/snipe/snipe-it/blob/master/.htaccess#L7 is a ideal fix.


<blockquote><img src="https://repository-images.githubusercontent.com/14346979/164a6580-7d79-11e9-865f-eae87593e11a" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/snipe/snipe-it">snipe-it/.htaccess at master · snipe/snipe-it</a></strong></div><div>A free open source IT asset/license management system - snipe-it/.htaccess at master · snipe/snipe-it</div></blockquote>

**Comment by snipe** [comment]
@bby-bishopclark I've had a PR open for 3 days for this. https://github.com/snipe/snipe-it/pull/11072 - just waiting on someone who uses apache to test.

**Comment by bby-bishopclark** [comment]
is THAT all?!?  Let me check my dev box.

Good news and bad news:
```
# rpm -qa snipe-it httpd
snipe-it-5.3.10-2.el8.noarch
httpd-2.4.37-43.module+el8.5.0+747+83fae388.3.x86_64
```

Apache here, but I can't get a newer snipe-it into the system until I update the package, and there was a minor change after 5.3.10 that is hosing the build test;  ownership or something.  It's only a coupla days, maybe, but it's gonna be a coupla days.

**Comment by snipe** [comment]
@bby-bishopclark just noticed this - where is that snipe-it RPM coming from? We don't build an RPM for this, so curious where you found that or if you're building it yourself?

**Comment by bby-bishopclark** [comment]
> @bby-bishopclark just noticed this - where is that snipe-it RPM coming from? 

We talked about this before.  An RPM is not on the project's radar, but crucial for adoption at my site, so yeah.  I used to do it full-time for OpenLinux, so it was almost no big deal.

**Comment by snipe** [comment]
I don't recall talking to you about it, but I believe you. I talk to a lot of people here. Literally thousands.

**Comment by snipe** [comment]
So if nobody is willing to test this, I can't merge it. I closed the PR, but will re-add if anyone can confirm this works.

**Comment by vapier** [comment]
on the other hand, you can't break more what is already broken

**Comment by bby-bishopclark** [comment]
Sorry, but I can't test.  It appears v6 made a dependency leap and leapt right out of EL7 support.  My packaging effort - and thus upgrades and thus testing - is stalled until I replatform on another EL that has what the project needs.   It's probably EL9 since its 10-year support window is only 10% used.  TL;DR is I have no v6 installs for the near term.

**Comment by scranfor** [comment]
I tested on Apache 2.4.41 as shipped in Ubuntu 20.04. Seems to work fine.

**Comment by vapier** [comment]
commit 157944b77422b3af61f7c3ff9557387940275cd9 fixed it for me with 2.4.54
